.github/workflows/reviewdog.yml

```diff
@@ -30,4 +30,27 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           reporter: github-pr-check
-          level: warning
\ No newline at end of file
+          level: warning
+          spelling:
+  lintApp:
+    name: Lint App
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: eviewdog/action-eslint@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          reporter: github-pr-check
+          workdir: "./app"
+          eslint_flags: "\"{src,apps,libs,test}/**/*.ts\""
```

Comment:
will this include tsx?

Reply:
I suspect it won't include anything and will just fail miserably